src/lib/operators/join_sort_merge/radix_cluster_sort.hpp

```diff
@@ -345,10 +321,15 @@ class RadixClusterSort {
     ColumnMaterializer<T> right_column_materializer(!_equi_case, _materialize_null_right);
     auto materialization_left = left_column_materializer.materialize(_input_table_left, _left_column_id);
     auto materialization_right = right_column_materializer.materialize(_input_table_right, _right_column_id);
-    auto materialized_left_segments = std::move(materialization_left.first);
-    auto materialized_right_segments = std::move(materialization_right.first);
-    output.null_rows_left = std::move(materialization_left.second);
-    output.null_rows_right = std::move(materialization_right.second);
+    auto materialized_left_segments = std::move(std::get<0>(materialization_left));
+    auto materialized_right_segments = std::move(std::get<0>(materialization_right));
```

Comment:
structured binding?

Reply:
Could you please take a look at 15f9797? I'd like to hear from someone who understands C++ that these changes do not have a negative impact performance.
I did not measure a difference, but we just might lack the right benchmarks.

Reply:
Looks like you are using `left_column_materializer.materialize` twice. I wouldn't expect any differences though.